.github/workflows/backport.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     name: Backport
     steps:
       - name: Backport Bot
-        if: contains( join(github.event.pull_request.labels.*.name), 'backport')
+        if: github.event.pull_request.merged && ( ( github.event.action == 'closed' && contains( github.event.pull_request.labels.*.name, 'backport') ) || contains( github.event.label.name, 'backport' ) )
```

Comment:
```suggestion
        if: github.event.pull_request.merged && ( ( github.event.action == 'closed' && contains( github.event.pull_request.labels.*.name, 'backport') ) || github.event.label.name == 'backport' )
```

Reply:
am I right?

Reply:
Shouldn't this be "startswith" or similar?

Reply:
ah sorry, yes!

Reply:
Are we sure merged is true when you actually merge it?

Reply:
After your merge the doc bot created a commit. I think that's a good hint it worked for the doc bot.

Reply:
Actually it should be `contains an element that startsWith backport` instead of `contains an element which is backport`